frontend/src/pages/Common/History/HistorySearch.js

```diff
@@ -61,7 +61,7 @@ const HistorySearch = ({ classes, fetchFirstHistoryEvents, users, eventTypes })
       <DatePicker
         id="filter-startat"
         className={classes.datepicker}
-        label={"start"}
+        label={strings.history.startDate}
         name="startAt"
         datetime={startAt}
```

Comment:
`strings.history.startDate` is missing in all language files

Reply:
since without a word in this area, a style bug occurs (the position of the date delete button is not accurate). This style bug should be resolved after setting `strings.history.startDate`